packages/now-go/test/fixtures/16-custom-flag/index.go

```diff
@@ -3,9 +3,11 @@ package handler
 import (
 	"fmt"
 	"net/http"
+
+	"custom-flag/custom"
```

Comment:
Is the indentation here correct?

Reply:
@styfle yes, I think so. because it format by `go-fmt`